templates/authorization/deploymentTopLayer.json

```diff
@@ -21,12 +21,12 @@
                     "IoTEdgeAPIProxy": {
                         "settings": {
                             "image": "mcr.microsoft.com/azureiotedge-api-proxy:1.0",
-                            "createOptions": "{\"HostConfig\": {\"PortBindings\": {\"8000/tcp\": [{\"HostPort\":\"8000\"}]}}}"
+                            "createOptions": "{\"HostConfig\": {\"PortBindings\": {\"443/tcp\": [{\"HostPort\":\"443\"}]}}}"
```

Comment:
Why is the port mapping needed on registry module?

Reply:
What do you mean? This is API proxy.
Registry module is still 5000

Reply:
oh I see, why is it map to 5000? It is not needed. I suppose it was added as a convenience. 
It is not ISA95 so it has no impact here.

Reply:
ok I can remove the mapping. To be perfectly consistent we need to remove it in the IIOT report too.